Model/Seller/DataProvider.php

```diff
@@ -0,0 +1,14 @@
+<?php
+namespace Smile\Seller\Model\Seller;
```

Comment:
Comment missing

/**
 * DISCLAIMER
 *
 * Do not edit or add to this file if you wish to upgrade this module to newer
 * versions in the future.
 *
 * @category  

 * @package   
 * @author   
 * @copyright 2019 Smile
 * @license   Open Software License ("OSL") v. 3.0
 */